### PR TITLE
Reviewing Lines Part 5

### PR DESCRIPTION
**Pull request by ttgmichael**
Dolar signs on QUESTLV100 lines 1600-2000 with some revisions.

Let me know your thoughts! :)

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -1667,165 +1667,165 @@ QUEST_LV_0100_20150317_001666	Believer Virgis
...
+QUEST_LV_0100_20150317_001677	$This is enough. Well done.{nl}I feel happy to see meat.
+QUEST_LV_0100_20150317_001678	$So... We have everything we needed... I will check the list again and report to Selina.{nl}Please wait a moment...{nl}
+QUEST_LV_0100_20150317_001679	$Ah... They're worse than those Vubbes. For real!! The potion jars are all missing...{nl}Those Zigris sometimes steal the objects that seem unique to them.{nl}
+QUEST_LV_0100_20150317_001680	$They are more than just thieves.{nl}If anything looks good, they'll take it away.
+QUEST_LV_0100_20150317_001681	$We don't have to worry about Zigris for a while now.{nl}They will be scared just thinking about you.{nl}
+QUEST_LV_0100_20150317_001682	$We have everything we need now.{nl}I was able to prepare them by the deadline.{nl}I am relieved now.
+QUEST_LV_0100_20150317_001683	$Fortunately, everything look alright.{nl}Those Zigrises... I want to show them how scary I am.
+QUEST_LV_0100_20150317_001684	$The Vubbes installed a flag at their camp not so long ago.{nl}Now they've been driven out and the Zigris have settled in.{nl} This isn't super important, but what is important is..{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1684]: installed -> planted/set up ?

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -1667,165 +1667,165 @@ QUEST_LV_0100_20150317_001666	Believer Virgis
...
+QUEST_LV_0100_20150317_001707	$Please do not forget your valuable experiences here.{nl}Don't be afraid, please proceed.
+QUEST_LV_0100_20150317_001708	$You woke me up to take the test.
+QUEST_LV_0100_20150317_001709	$I've been waiting for you. I'm the Swordsman Master, the one who will help you.{nl}I'm not sure if you know about attribute training. {nl}
+QUEST_LV_0100_20150317_001710	$Attributes can give additional abilities to stats and skills. {nl}You can learn attributes from each class master by paying them silver.
+QUEST_LV_0100_20150317_001711	$You'll be able to fight easier if you learn attributes. {nl}
+QUEST_LV_0100_20150317_001712	$You've come to the right place. I'm the Wizard Master. {nl}I'm going to teach you about attribute training. {nl}
+QUEST_LV_0100_20150317_001713	$Attributes can give additional abilities to stats and skills. {nl}You can learn attributes from each class master by paying them silver. 
+QUEST_LV_0100_20150317_001714	$Are the info on attributes crystal-clear? Now you'll be able to fight the enemies easier.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1714]: Are the info?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -1667,165 +1667,165 @@ QUEST_LV_0100_20150317_001666	Believer Virgis
...
+QUEST_LV_0100_20150317_001715	$How you become strong depends on what choice you make. {nl}It differs depending on your choice and thoughts.
+QUEST_LV_0100_20150317_001716	$Good to see you. I am the Archer Master. {nl}Do you know about attribute training? {nl}
+QUEST_LV_0100_20150317_001717	$Attributes can give additional abilities to stats and skills. {nl}You can learn attributes from each class master by paying them silver. 
+QUEST_LV_0100_20150317_001718	$Mastering these attributes will help your skills in combat.
+QUEST_LV_0100_20150317_001719	$Will is the force that creates magic power. {nl}For a Wizard, there is no stronger weapon.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1719]: Willpower -> Will? Why the change?

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -1667,165 +1667,165 @@ QUEST_LV_0100_20150317_001666	Believer Virgis
...
+QUEST_LV_0100_20150317_001731	$Plantain Grass are all around here so you can easily find them.{nl}I'll be working on something else.
+QUEST_LV_0100_20150317_001732	$There are many Tinis when you climb up the hill from the road down.{nl}8 big scoops will do.
+QUEST_LV_0100_20150317_001733	$We don't have time collect the potion jars again.{nl}Let's steal back the jars that Zigris stole from us. They are the ones who started this.
+QUEST_LV_0100_20150317_001734	$Monsters really like the Soul Crystal.{nl}They become strong when they absorb its power.
+QUEST_LV_0100_20150317_001735	$If something goes wrong... then go meet the Gatekeeper Owl.{nl}He is most clever among us.{nl}May the goddess bless you.
+QUEST_LV_0100_20150317_001736	$Nice to meet you, follower of the goddess. {nl}Do you know about learning attributes?
+QUEST_LV_0100_20150317_001737	$Welcome, you who follow the goal of the goddess.{nl}Do you know about training attributes?
+QUEST_LV_0100_20150317_001738	$Now do you grasp what attributes are? {nl} Acquiring attributes will help you to easily defeat your enemies. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1738]: Acquiring -> Learning

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -1667,165 +1667,165 @@ QUEST_LV_0100_20150317_001666	Believer Virgis
...
+QUEST_LV_0100_20150317_001787	$So you guys are the ones who wanted go to the battlefield?{nl}You guys have no skills. There's little to expect from you guys.{nl}And those pets... are you guys crazy enough to take those pets to the perilous battlefield?
+QUEST_LV_0100_20150317_001788	$What's with those defiant glares? {nl}You guys don't wanna accept that you're not skillful?
+QUEST_LV_0100_20150317_001789	$Okay. I like your attitudes.{nl}But that is not enough.{nl}Defeat those Golems at the training camp and bring me the evidence that you defeated them. Then I will acknowledge your skill.
+QUEST_LV_0100_20150317_001790	$Fighting Golems will be easier than going to the battlefield.
+QUEST_LV_0100_20150317_001791	$Lads. You came. Show me the evidence.{nl}You defeated Golems? or you just ran way from them?
+QUEST_LV_0100_20150317_001792	$Good! Very nice. I will acknowledge your skills.{nl}Don't ever forget. The ones who survive in this war are the winners.{nl}
+QUEST_LV_0100_20150317_001793	$Okay. Return back to Raimondas.{nl}Be confident.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1793]: Be confident -> Have confidence. (Have confidence in yourself) 

?

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:1714]: Been trying to diversify this line as it's literally used in multiple ways.

**Comment by Sunaries** [comment]
Good to merge. :+1: 

(Great job on these PRs)

**Comment by imcgames** [comment]
Thank you :+1:
